subsys/bluetooth/host/gatt.c

```diff
@@ -2116,8 +2120,8 @@ static int gatt_indicate(struct bt_conn *conn, uint16_t handle,
 
 	len = sizeof(*ind) + params->len;
 
-	req = gatt_req_alloc(gatt_indicate_rsp, params, NULL,
-			     BT_ATT_OP_INDICATE, len);
+	req = gatt_req_alloc(gatt_indicate_rsp, params, NULL, BT_ATT_OP_INDICATE, len,
+			     params->bearer_option);
```

Comment:
if CONFIG_BT_EATT is not enabled the compilation will fail here and in similar places